packages/datagateway-search/src/card/investigationSearchCardView.component.tsx

```diff
@@ -337,54 +467,120 @@ const InvestigationCardView = (
+                    -1
                   }
                 />
               </ActionButtonDiv>
             ),
           ]
         : [],
-    [data, hierarchy, sizeQueries]
+    [hierarchy, aggregatedIds, sizeQueries, paginatedSource]
   );
 
   return (
-    <CardView
-      data={data ?? []}
-      totalDataCount={totalDataCount ?? 0}
-      onPageChange={pushPage}
-      onFilter={pushFilter}
-      onSort={handleSort}
-      onResultsChange={pushResults}
-      loadedData={!dataLoading}
-      loadedCount={!countLoading}
-      filters={filters}
-      sort={sort}
-      page={page}
-      results={results}
-      title={title}
-      description={description}
-      information={information}
-      moreInformation={moreInformation}
-      buttons={buttons}
-    />
+    <Grid
+      data-testid="investigation-search-card-view"
+      container
+      spacing={1}
+      sx={{ height: '100%' }}
+    >
+      <Grid item xs={2} sx={{ height: '100%' }}>
+        {data?.pages && (
+          <FacetPanel
+            allIds={aggregatedIds}
+            entityName="Investigation"
+            showApplyButton={haveUnappliedFilters}
+            facetClassification={facetClassificationFromSearchResponses(
+              data.pages
+            )}
+            selectedFacetFilters={selectedFacetFilters}
+            onAddFilter={(dimension, filterValue) =>
+              addFacetFilter({
+                dimension,
+                filterValue,
+                applyImmediately: false,
+              })
+            }
+            onRemoveFilter={(dimension, filterValue) =>
+              removeFacetFilter({
+                dimension,
+                filterValue,
+                applyImmediately: false,
+              })
+            }
+            onApplyFacetFilters={() => {
+              applyFacetFilters();
+              refetch();
+            }}
+          />
+        )}
+      </Grid>
+      <Grid item xs={10}>
+        <SelectedFilterChips
+          filters={filters}
+          onRemoveFilter={removeFilterChip}
+        />
+        <Paper variant="outlined" sx={{ height: '100%', marginTop: 1 }}>
+          <div>
+            {aborted ? (
+              <Paper>
+                <Typography align="center" variant="h6" component="h6">
+                  {t('loading.abort_message')}
+                </Typography>
+              </Paper>
+            ) : (
+              <CardView
+                entityName="Investigation"
+                data={paginatedSource ?? []}
+                totalDataCount={
+                  aggregatedIds?.length + (hasNextPage ? 1 : 0) ?? 0
+                }
+                onPageChange={pushPage}
+                onFilter={pushFilter}
+                onSort={handleSort}
+                onResultsChange={pushResults}
+                loadedData={!isLoading}
+                loadedCount={!isLoading}
+                filters={filters}
```

Comment:
Need to remove passing of filters to `investigationSearchCardView` as well

Reply:
Oops, forgot about it.